target_color: 'red'
estimate_color: 'black'
agent_color: 'blue'
xmin: -2.0
xmax: 2.0
ymin: -2.0
ymax: 2.0
